extensions/mssql/notebooks/TSG/cluster-status.ipynb

```diff
@@ -90,7 +85,7 @@
         },
         {
             "cell_type": "code",
-            "source": "# Display status of controller\nresults = !mssqlctl bdc control status show\nshow_results(results)",
+            "source": "# Display status of controller\nresults = !azdata bdc control status show\nshow_results(results)",
```

Comment:
While it's not needed in this PR we need to move away from HTML tables -> Markdown tables. Just above this cell is a table that doesn't render in Github. We should prefer Markdown wherever possible. If you can engage with PM on this, let's try to get this resolved soon.

Reply:
Updated it.